css-two-column-layout/styles.css

```diff
@@ -0,0 +1,96 @@
+* {
+  box-sizing: border-box;
+}
+
+h1 {
+  text-align: center;
+  font-family: Karla, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 42px;
+}
+
+h5 {
+  font-family: Karla, sans-serif;
+  font-style: normal;
+  font-weight: 900;
+}
+
+img {
+  width: 100px;
+  height: auto;
+  padding: 0 5px;
+}
+
+p {
+  font-family: Karla, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 14px;
+  line-height: 22px;
+}
+
+button {
+  font-family: Karla, sans-serif;
+  font-size: 14px;
+  font-weight: bold;
+  color: lightcoral;
+  background-color: white;
+  border: 2px solid lightcoral;
+  padding: 8px 38px;
+  margin-top: 16px;
+}
+
+.container {
+  margin: 50px 100px;
+}
```

Comment:
You should change the styling here to include a `max-width` and `margin: 50px auto`